athenet/sparsifying/derest/activation.py

```diff
@@ -0,0 +1,27 @@
+"""For any neuron, we calculate its range of activation for inputs of neural
+networks in given range. For simplicity, we assume it is [0, 255]. We do it
```

Comment:
"We do it"
Too many shortcuts. It is the moment you should use "calculate" again, or some synonym of it.